src/servers/grpc_server.cc

```diff
@@ -679,26 +679,26 @@ InferResponseAlloc(
         *actual_memory_type_id = pr->second.device_id_;
         use_shm = true;
 
-        LOG_VERBOSE(1) << "GRPC shared-memory: " << tensor_name << ", size "
-                       << byte_size << ", addr " << *buffer;
+        LOG_VERBOSE(1) << "GRPC: using shared-memory for " << tensor_name
+                       << ", size " << byte_size << ", addr " << *buffer;
       }
     }
 
     if (!use_shm) {
       // Can't allocate for any memory type other than CPU. If asked to
       // allocate on GPU memory then force allocation on CPU instead.
       if (*actual_memory_type != TRTSERVER_MEMORY_CPU) {
-        LOG_VERBOSE(1) << "GRPC allocation failed for type "
-                       << *actual_memory_type << " for " << tensor_name
-                       << ", will use type " << TRTSERVER_MEMORY_CPU;
+        LOG_VERBOSE(1)
+            << "GRPC: unable to provide " << tensor_name
+            << " in requested memory type, will use type TRTSERVER_MEMORY_CPU";
```

Comment:
Since this is only triggered for GPU, it means we can replace  requested memory type with TRTSERVER_MEMORY_GPU